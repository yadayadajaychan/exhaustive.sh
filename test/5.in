1A 2B 1C
