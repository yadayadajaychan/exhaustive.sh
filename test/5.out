A=1'd0; B=2'd0; C=1'd0; #10;
A=1'd0; B=2'd0; C=1'd1; #10;
A=1'd0; B=2'd1; C=1'd0; #10;
A=1'd0; B=2'd1; C=1'd1; #10;
A=1'd0; B=2'd2; C=1'd0; #10;
A=1'd0; B=2'd2; C=1'd1; #10;
A=1'd0; B=2'd3; C=1'd0; #10;
A=1'd0; B=2'd3; C=1'd1; #10;
A=1'd1; B=2'd0; C=1'd0; #10;
A=1'd1; B=2'd0; C=1'd1; #10;
A=1'd1; B=2'd1; C=1'd0; #10;
A=1'd1; B=2'd1; C=1'd1; #10;
A=1'd1; B=2'd2; C=1'd0; #10;
A=1'd1; B=2'd2; C=1'd1; #10;
A=1'd1; B=2'd3; C=1'd0; #10;
A=1'd1; B=2'd3; C=1'd1; #10;
